#!/usr/bin/env python

# jplaced
# 2022, Universidad de Zaragoza

PACKAGE = "frontier_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("ig_threshold", double_t, 0, "A double parameter", 0.5, 0, 5)

exit(gen.generate(PACKAGE, "frontier_detector", "informationGain"))
